Utilities/DatabasePorter/api.go

```diff
@@ -130,7 +130,7 @@ func GetEntry(hash string) (interfaces.IEBEntry, error) {
 		}
 		return entry, nil
```

Comment:
You only return if there is no error. If there is an error, you will continue and skip the return.